### PR TITLE
Improve introduction section, add user guide section

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+************
+Introduction
+************
+
+Overview
+========
+
+``Remme`` is a blockchain-based protocol used for issuing and management of X.509 client certificates to resolve issues
```
Comment (eugene-babichenko) [inline docs/introduction.rst]: Not X.509 certs, public keys in general.

File: docs/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+related to cybersecurity, IoT connectivity, data integrity, digital copyright protection, transparency etc.
+
+``Remme core`` is built on Hyperledger Sawtooth platform, allowing to be flexible in language choice during the
+development process. ``Remme core`` supports JS, .NET, that’s why you to easily embed protocol in your project.
```
Comment (eugene-babichenko) [inline docs/introduction.rst]: Not true. It exposes the RPC API which is wrapped with a bunch of integration libraries (JS, .NET, etc)

Comment (dmytrostriletskyi) [reply docs/introduction.rst]: @eugene-babichenko, done.

File: docs/introduction.rst (hunk 1 above)
Comment (dmytrostriletskyi) [reply docs/introduction.rst]: @eugene-babichenko, done.

**Pull request merged**